barretenberg/ts/src/crs/node/index.ts

```diff
@@ -41,9 +43,12 @@ export class Crs {
 
     this.logger(`Downloading CRS of size ${this.numPoints} into ${this.path}`);
     const crs = new NetCrs(this.numPoints);
-    await crs.init();
-    writeFileSync(this.path + '/bn254_g1.dat', crs.getG1Data());
-    writeFileSync(this.path + '/bn254_g2.dat', crs.getG2Data());
+    const [g1, g2] = await Promise.all([crs.streamG1Data(), crs.streamG2Data()]);
+
+    await Promise.all([
+      finished(Readable.fromWeb(g1 as any).pipe(createWriteStream(this.path + '/bn254_g1.dat'))),
```

Comment:
The `as any` is needed to convert between fetch and node's web stream :disappointed: